N24DCCN162/n24dccn162_buoi5/bai_2.cpp

```diff
@@ -0,0 +1,16 @@
+#include<iostream>
+using namespace std;
+int main() {
+    int so;
+    cout<<"Hay nhap 1 so tu 2 den 9: ";
+    cin>>so;
+    if((so<2)||(so>9)){

```

Comment:
Đề yêu cầu chỉ cần in bảng cửu chương với n nhập từ bàn phím, ko đề cập là n phải từ 2 tới 9